tests/unittests/tiglWing.cpp

```diff
@@ -409,6 +411,39 @@ TEST_F(TiglWing, depthDirection)
     vtp.SetSymmetryAxis(vtpSymAx);
 }
 
+TEST_F(TiglWing, testGetWingWithCutOuts)
+{
+    const char* filename = "TestData/simpletest.cpacs.xml";
+    ReturnCode tixiRetX;
+    TiglReturnCode tiglRetX;
+
+    TixiDocumentHandle tiglHandleX = -1;
+    TiglCPACSConfigurationHandle tixiHandleX = -1;
+
+    tixiRetX = tixiOpenDocument(filename, &tixiHandleX);
+    ASSERT_TRUE (tixiRetX == SUCCESS);
+    tiglRetX = tiglOpenCPACSConfiguration(tixiHandleX, "", &tiglHandleX);
+    ASSERT_TRUE(tiglRetX == TIGL_SUCCESS);
+
+    tigl::CCPACSConfigurationManager& manager = tigl::CCPACSConfigurationManager::GetInstance();
+    tigl::CCPACSConfiguration& config         = manager.GetConfiguration(tiglHandleX);
+    tigl::CTiglUIDManager& uidmgr = config.GetUIDManager();
+    auto& wing = uidmgr.ResolveObject<tigl::CCPACSWing>("Wing");
+
+    // Check for warning if no control surfaces are defined
+      { // Scope to destroy object of type CaptureTiGLLog and therefore reset console verbosity
+          CaptureTiGLLog t{TILOG_WARNING};
+          wing.GetLoftWithCutouts();
+          auto logOutput = t.log();
+
+          std::string comparisonString = "No control devices defined, GetLoftWithCutOuts() will return a clean shape.";
+          ASSERT_TRUE((logOutput.find(comparisonString)) != std::string::npos);
+      } // scope
+
+    ASSERT_TRUE(BRepCheck_Analyzer(wing.GetLoftWithCutouts()).IsValid());
+
+}
+

```

Comment:
You are using a Test fixture `TiglWing` here which loads the D150 into the static variables `tixiHandle` and `tiglHandle` and then overwrite these handles with the simpletest. This might be probablamatic if you run other unit tests from the same fixture afterwards, that expect the D150. 

In the same file `tiglWing.cpp` there is another text fixture called `WingSimple`, that loads the simpletest.cpacs.xml already and stores it in `tixiSimpleWingHandle` and `tiglSimpleWingHandle`. So why not just use this fixture instead?

Reply:
Oh yes, I missed that, thank you. I already changed it.